src/actions/ApplicationActions.js

```diff
@@ -34,7 +34,9 @@ export function getApplicationState(done) {
         }
         return api.form().then((r) => {
           formData = r.data
-          dispatch(updateApplication('Settings', 'formType', formData.Metadata.form_type))
+          const formType = window.formType ? window.formType : formData.Metadata.form_type
```

Comment:
If there's a URL param `formType`, it always has higher priority than the `formType` that comes with the form data.